### PR TITLE
CustomId console, reserved project

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Prevent creating a project with `console` as ID. making `console` a reserved project ID

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

File: app/controllers/api/projects.php [hunk 1]
```diff
@@ -77,6 +77,9 @@
         }
 
         $projectId = ($projectId == 'unique()') ? $dbForConsole->getId() : $projectId;
+        if($projectId === 'console') {
```
Comment (Meldiron) [inline app/controllers/api/projects.php:80]: Could you double-check if this is as string elsewhere, or do we have const for it?

Comment (lohanidamodar) [reply app/controllers/api/projects.php:80]: we are using similar thing in other places, so I don't think we have a constant for this
https://github.com/appwrite/appwrite/blob/9e5436f80d88f62b48c6c2c9d87dd31169a8058a/app/init.php#L742

Comment (Meldiron) [reply app/controllers/api/projects.php:80]: Awesome, thanks.

**Review by Meldiron — APPROVED** [review]
LGTM

**Review by eldadfux — APPROVED** [review]

**Comment by TorstenDittmann** [comment]
It's not really a reserved keyword, but the project in this case actually already exists right?

**Comment by lohanidamodar** [comment]
> It's not really a reserved keyword, but the project in this case actually already exists right?

it does, but I don't get back duplicate project error, I get error on developer tools, and when I refresh, console project is listed along with my other projects. I can't create collections, buckets but I can create keys, webhooks, add platforms etc.

**Pull request merged**